### PR TITLE
update link in pipeline check

### PR DESCRIPTION
**Pull request by bagxi**

### CODE REVIEW

File: bin/_check_pipeline.sh [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+function gdrive_download () {
+  CONFIRM=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate "https://docs.google.com/uc?export=download&id=$1" -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')
+  wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$CONFIRM&id=$1" -O $2
+  rm -rf /tmp/cookies.txt
+}
+
 mkdir -p data
 
-wget https://www.dropbox.com/s/8aiufmo0yyq3cf3/ants_bees_cleared_190806.tar.gz
+gdrive_download 1czneYKcE2sT8dAMHz3FL12hOU7m1ZkE7 ants_bees_cleared_190806.tar.gz
```
Comment (TezRomacH) [inline bin/_check_pipeline.sh]: let's use `bash bin/scripts/download-gdrive 1czneYKcE2sT8dAMHz3FL12hOU7m1ZkE7 ants_bees_cleared_190806.tar.gz` to remove code duplications

File: bin/_check_pipeline.sh [hunk 2]
```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env bash
 set -e
 
+function gdrive_download () {
+  CONFIRM=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate "https://docs.google.com/uc?export=download&id=$1" -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')
```
Comment (TezRomacH) [inline bin/_check_pipeline.sh]: also, could you please check `sed` func in `bash bin/scripts/download-gdrive`?

On macOS i got
```bash
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

File: bin/_check_pipeline.sh (hunk 1 above)
Comment (TezRomacH) [reply bin/_check_pipeline.sh]: oh, it's not the Catalyst repo 😄 

so, it's getting easier: just `download-gdrive 1czneYKcE2sT8dAMHz3FL12hOU7m1ZkE7 ants_bees_cleared_190806.tar.gzz

File: bin/_check_pipeline.sh (hunk 2 above)
Comment (bagxi) [reply bin/_check_pipeline.sh]: on Ubuntu16.04 all fine :)

File: bin/_check_pipeline.sh (hunk 1 above)
Comment (bagxi) [reply bin/_check_pipeline.sh]: according to `requirements/requirements_local.txt` we have `catalyst==19.10` requirement. so, either we should update it to `catalyst==19.11.1` or leave it as is.

Comment (TezRomacH) [reply bin/_check_pipeline.sh]: I suggest increasing the version to `19.11.1`

**Pull request merged**